bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -218,6 +218,62 @@ impl Script {
         }
     }
 
+    /// Checks whether a script pubkey is a bare multisig output.
+    #[inline]
+    pub fn is_multisig(&self) -> bool {
+        if self.0.is_empty() {
+            return false;
+        }
+
```

Comment:
Not strictly required because we catch this case below on the first call to `instructions.next()`.

Reply:
Updated in de5103b9d6e3c2acd970da6e5b6036ca16c690cb